src/main/java/wolox/training/web/dtos/deserializers/IsoLocalDateDeserializer.java

```diff
@@ -0,0 +1,32 @@
+package wolox.training.web.dtos.deserializers;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * A {@link com.fasterxml.jackson.databind.JsonDeserializer} to create {@link LocalDate} instances
+ * from ISO 8601 format {@link String}s.
+ */
+public class IsoLocalDateDeserializer extends StdDeserializer<LocalDate> {
+
```

Comment:
Why do you use deserializer and serializer class?  maybe an annotation is more simple to code

Reply:
Yes, I just discovered ```@JsonFormat(pattern = "yyyy-MM-dd")```.
I also discovered that it also works without setting any annotation, which default to ```yyyy-MM-dd```.